applications/TopologyOptimizationApplication/tests/Small_Cantilever/ProjectParameters.json

```diff
@@ -0,0 +1,123 @@
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0,
+        "start_time"      : 0.0,
+        "end_time"        : 1.0
+    },
+    "solver_settings"          : {
+        "solver_type"                        : "topology_optimization_simp_static_solver",
+        "model_part_name"                    : "Structure",
+        "domain_size"                        : 3,
+        "echo_level"                         : 0,
+        "analysis_type"                      : "linear",
+  	"problem_domain_sub_model_part_list" : ["Parts_structure"],
+        "processes_sub_model_part_list"      : ["DISPLACEMENT_disp","PointLoad3D_load"],
+        "model_import_settings"              : {
+            "input_type"     : "mdpa",
+            "input_filename" : "Small_Cantilever_Hex"
+        },
+        "material_import_settings"           : {
+            "materials_filename" : "StructuralMaterials.json"
+        },
+        "time_stepping"                      : {
+            "time_step" : 1.1
+        },
+        "line_search"                        : false,
+        "convergence_criterion"              : "residual_criterion",
+        "displacement_relative_tolerance"    : 0.0001,
+        "displacement_absolute_tolerance"    : 1e-9,
+        "residual_relative_tolerance"        : 0.0001,
+        "residual_absolute_tolerance"        : 1e-9,
+        "max_iteration"                      : 10,
+        "rotation_dofs"                      : false,
+        "compute_reactions"                  : true,
+        "reform_dofs_at_each_step"           : false,
+        "move_mesh_flag"                     : true,
+        "linear_solver_settings"             :  {
+            "solver_type"         : "amgcl",
+            "max_iteration"       : 500,
+            "tolerance"           : 1e-8,
+            "scaling"             : false
+        } 
+               
+
+    },
+    "processes"        : {
+   	"constraints_process_list" : [{
+		"python_module" : "assign_vector_variable_process",
+        	"kratos_module" : "KratosMultiphysics",
+        	"Parameters"    : {
+                "mesh_id"         : 0,
+        	    "model_part_name" : "Structure.DISPLACEMENT_disp",
+        	    "variable_name"   : "DISPLACEMENT",
+                "constrained"     : [true,true,true],
+        	    "value"           : [0.0,0.0,0.0]
+        	}
+           }],
+        "loads_process_list"       : [{
+            "python_module" : "assign_vector_by_direction_to_condition_process",
+            "kratos_module" : "KratosMultiphysics",
+            "check"         : "DirectorVectorNonZero direction",
+            "Parameters"    : {
+           	    "mesh_id"         : 0,
+           	    "model_part_name" : "Structure.PointLoad3D_load",
+            	"variable_name"   : "POINT_LOAD",
+            	"modulus"         : "1",
+            	"direction"       : [0.0,1.0,0.0]
+            }
+    }],
+    "output_configuration"     : {
```

Comment:
you should use the `output_processes` for this (BTW a test should not write any unnecessary output at all to make it as fast as possible)
=> if you want to leave it in you can comment it (same way as C++ comments)

Reply:
I will remove this part in the future and make the optimization process equivalent to the process within structural mechanics